lib/pages/whiteboard/whiteboard_page.dart

```diff
@@ -39,11 +40,12 @@ class _WhiteboardPageState extends State<WhiteboardPage> {
                   ),
                   StyledButton(
                     onPressed: () async {
-                        await widget.addMessage(_controller.text);
+                      var text = _controller.text;
                       _controller.clear();
+                      await widget.addMessage(text);
```

Comment:
Doesn't this pass a future to the function that wants a String? Or is flutter smart about this